packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1785,6 +1787,8 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     _fadeoutTimer?.cancel();
     _fadeoutAnimationController.forward();
     _dragScrollbarAxisOffset = localPosition;
+    _thumbPressVertical = localPosition.dy - scrollbarPainter._thumbOffset;
+    _thumbPressHorizontal = localPosition.dx - scrollbarPainter._thumbOffset;
```

Comment:
Do we need to keep track of both the horizontal and the vertical? Or just the axis we care about? It looks like you filter for it later in `handleThumbPressUpdate`

Reply:
Great idea, I cleaned this up. 

But for some reason, the golden file change message is thrown. This shouldn't have any golden changes AFAIK.